apps/full-site-editing/full-site-editing-plugin/full-site-editing/blocks/site-logo/edit.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * External dependencies
+ */
+import { IconButton, ServerSideRender, Toolbar } from '@wordpress/components';
+import { Fragment } from '@wordpress/element';
+import { BlockControls } from '@wordpress/editor';
+import { __ } from '@wordpress/i18n';
+import { addQueryArgs } from '@wordpress/url';
+
+function SiteLogoEdit( { className } ) {
+	const siteIdentityUrl = addQueryArgs( 'customize.php', {
+		'autofocus[section]': 'title_tagline',
+		return: window.location.href,
+	} );
+
+	return (
+		<Fragment>
```

Comment:
```suggestion
		<>
```

Reply:
Does Calypso build tooling support this shorthand?

Reply:
Since this shorthand is provided by React and [Calypso externalizes that dependency](https://github.com/Automattic/wp-calypso/blob/dcde4fd710dfc6ba1092491456ce90fe35e8151b/packages/wordpress-external-dependencies-plugin/index.js#L29-L31), we shouldn't worry about using it as long as the build is loaded by an environment where React v16.2.0 is available ([fragments were included in v16.2.0](https://reactjs.org/blog/2017/11/28/react-v16.2.0-fragment-support.html)).

Given that WordPress is currently using [React v16.8.4](https://github.com/WordPress/gutenberg/blob/8589bc1733359aa5cf38dafad195d54c4616b102/lib/client-assets.php#L426-L442), we can definitely use fragments and their shorthands.